tests/integration/test_package_managers.py

```diff
@@ -44,12 +42,54 @@
                 repo="https://github.com/cachito-testing/gomod-vendored.git",
                 ref="ff1960095dd158d3d2a4f31d15b244c24930248b",
                 packages=({"path": ".", "type": "gomod"},),
+                check_output_json=False,
+                check_deps_checksums=False,
+                check_vendor_checksums=False,
                 expected_rc=2,
                 expected_output='The "gomod-vendor" or "gomod-vendor-check" flag'
                 " must be set when your repository has vendored dependencies",
             ),
             id="gomod_vendored_without_flag",
         ),
+        # Test case checks if vendor folder with dependencies will remain unchanged in cloned
+        # source repo, deps/ folder in output folder should be empty.
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/gomod-vendored.git",
+                ref="ff1960095dd158d3d2a4f31d15b244c24930248b",
+                packages=({"path": ".", "type": "gomod"},),
+                flags=["--gomod-vendor"],
+                expected_rc=0,
+                expected_output="All dependencies fetched successfully",
+            ),
+            id="gomod_vendored_with_flag",
+        ),
+        # Test case checks if vendor folder will be created with dependencies in cloned
+        # source repo deps/ folder in output folder should be empty.
```

Comment:
I think a comma is missing here:
```suggestion
        # source repo, deps folder in output folder should be empty.
```
Nitpick: I would probably ommit the `/` here as well.